src/Core/Silk.NET.BuildTools/Bind/StructWriter.cs

```diff
@@ -503,6 +503,16 @@ public static void WriteStruct
                 sw.WriteLine();
             }
 
+            var handleField = @struct.Fields.FirstOrDefault(t => t.Name == "Handle" && t.Type.Name == "ulong");
```

Comment:
Good spot, but this isn't _quite_ the right place to put it as this will generate for every structure (including those that don't have a `Handle` field). What I decided to do in my PR is handle this in the `VulkanReader` i.e. right at the source of the generation of those `Handle` fields. But something at this layer would've worked to with a check for a structure being handle-like! Many ways to skin a cat :)